### DISCUSSION TITLE
Recognition difference between uncompressed sound and lossy compressed (mp3, ...)

### DISCUSSION THREAD

**Discussion by fherb2** [birdnet-team/BirdNET-Analyzer#243] [General]
Hello Stefan,

before the question: My setup settings:

1) Stereo-Mics -> USB-SoundCard -> RasPi_1 -> --audio-stream-- -> RasPi2~~> OPUS-Files (15 minutes slices) + parallel work: read into BirdNetPi for ad-hoc-analyzing.

2. ...4.) AudioMoth (https://www.openacousticdevices.info/audiomoth) -> WAV-Files

for 1): has been in used since last spring. 2...4) will start work at the end of February until July.

So I got and will get a lot of data what will be interesting to evaluate. (If anybody want to use this data: requests via: www.karlincam.cz .)

It is planned to use these setup over some years. I will get Terrabytes of uncompressed data. So the question is:

- Can compression (.mp3; I use instead .opus) reduce the recognition result?
- Are the learning data of the neural model coming form uncompressed sound files?
- What would you recommend: Is compression a problem for recognizing? Or: Where is a good level of compressing what would not be influence the recognition result?
- Or is there a official "good recommendations for compressing audio files" for use with BirdNet-Analyzer?

Best regards, Frank

**Comment by kahst** [comment]
We just recently had a similar discussion here: https://github.com/Mattk70/Chirpity-Electron/issues/33#issuecomment-1931955306

The gist is: BirdNET should be used to compression artifacts and results shouldn't vary too much. However, better quality will always yield better results. I would recommend experimenting with some quality settings before the deployment (e.g., strap a few recording units, each with different settings, to the same tree and then analyze the soundscapes to see how scores differ).

**Reply by fherb2** [reply]
Thanks, Stefan!

I will take the information into account accordingly in my recording configurations.

Best regards, Frank
